README.md

```diff
@@ -222,6 +222,14 @@ To make a printable version without any background images, add `--no-bg` flag
 tools/build.sh <LANGUAGE> --printable --no-bg
 ```
 
+## 📱 In-browser development using Codespaces
+
+If you have a Github account, you can select in-browser development option by clicking on the green `Code` button and then selecting `Codespaces` this will start a unique container with the whole development IDE called VS Code.
```

Comment:
```suggestion
If you have a GitHub account, you can select in-browser development option by clicking on the green `Code` button and then selecting `Codespaces` this will start a unique container with the whole development IDE called VS Code.
```